packages/venia-ui/lib/components/CartPage/ProductListing/product.css

```
@@ -27,9 +27,10 @@
     grid-area: details;
     grid-gap: 0.5rem;
     grid-template-areas:
-        'name .'
+        'name quantity'
```

I believe this is changing in #2577 to reflect a bit of overlooked UX intent.

```suggestion
        'name name'
```